Makefile

```diff
@@ -91,8 +91,11 @@ build-linux: go.sum
 	LEDGER_ENABLED=false GOOS=linux GOARCH=amd64 $(MAKE) build
 
 build-contract-tests-hooks:
-	mkdir -p $(BUILDDIR)
-	go build -mod=readonly $(BUILD_FLAGS) -o $(BUILDDIR)/ ./cmd/contract_tests
+ifeq ($(OS),Windows_NT)
```

Comment:
Hold on a sec please. ./cmd/contract_tests does not exist anymore. @shahankhatch please just remove this target from the Makefile all its references.

Reply:
cmd/contract_tests does still exist in the code for v2.0.x, should I still remove since we're continuing with v4+?

Reply:
It seems I didn't notice that this PR actually targets the 2.0 branch. Thus no, please go ahead and ignore my review.

Reply:
Actually, one thing must be said. I believe that the new reproducible build system does not support gaia v2.0. Thus, unless you managed to test the new build system successfully, this PR is IMvHO superfluous.

Reply:
I've tested the generated Mac reproducible v2.0.x binary, and the version prints out correctly. Since we're currently not testing the generated reproducible binaries in simulations I'll assume that they are as correct as the reproducible binaries generated on the v4.0.x branch.